### PR TITLE
allstar usecase(#21)

### PR DESCRIPTION
**Pull request by EjiroLaurelD**
Hello @jpkrohling 
I have made the recommend changes to my initial pr and also created a new branch. 
I wrote the document to outline how SIG security members use Allstar. Please let me know your thoughts on it.
Thank you for your time

### CODE REVIEW

File: sig-allstar.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## SIG Security's use of Allstar for the OpenTelemetry(OTEL) project
```
Comment (jpkrohling) [inline sig-allstar.md:1]:
```suggestion
## SIG Security's use of Allstar for the OpenTelemetry project
```

File: sig-allstar.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Allstar is a security policy engine that helps organizations automate and enforce security best practices. It can be used to scan code, dependencies, and infrastructure for vulnerabilities. It can also be used to enforce best practices for code reviews, security testing, and vulnerability management.
```
Comment (jpkrohling) [inline sig-allstar.md:3]: Can you add a link to allstar?

Comment (EjiroLaurelD) [reply sig-allstar.md:3]: sure, will include it right away

**Comment by EjiroLaurelD** [comment]
> I think the file name should be just `recommendation-allstar.md`. 
> 
> @cartersocha, @codeboten, I'm missing some context for this PR. Do we need this here at all? Or do we need a doc describing our specific allstar setup? 

Hello @jpkrohling @cartersocha @codeboten 
Any update on this please?

**Comment by jpkrohling** [comment]
Sorry, but after reviewing #21 again, I don't think it was meant to be an entry on the recommendations. From @codeboten's message on that issue:

> [Allstar](https://github.com/ossf/allstar) was proposed as a way to achieve consistency across the repositories in the org with regards to security policy. This issue is to:
> 
> * determine how much of the checklist allstar can cover
> * what items on the checklist above still need to be manually configured in individual repositories
> * propose the steps needed to enable allstar across the organization and open issues in the appropriate repositories
> * document the usage of allstar in the security sig repository

**Comment by codeboten** [comment]
Thanks for the work @EjiroLaurelD, I think the details you've captured here in this PR could be added to the original issue in a comment or as @jpkrohling in a separate google doc. 

Can this other [PR](https://github.com/open-telemetry/sig-security/pull/22) be closed in favour of [this current one](https://github.com/open-telemetry/sig-security/pull/29)?

**Comment by EjiroLaurelD** [comment]
> Thanks for the work @EjiroLaurelD, I think the details you've captured here in this PR could be added to the original issue in a comment or as @jpkrohling in a separate google doc.

Okay thank you, I will add my progress detail as a comment on the parent issue.

> 
> Can this other [PR](https://github.com/open-telemetry/sig-security/pull/22) be closed in favour of [this current one](https://github.com/open-telemetry/sig-security/pull/29)?

Yes please, we can close this [PR](https://github.com/open-telemetry/sig-security/pull/22).

**Pull request closed without merging**